app/views/users/_form_fields.html.erb

```diff
@@ -25,7 +25,7 @@
   </p>
 <% end %>
 
-<% if policy(User).assign_role? %>
+<% if policy(current_user).assign_role? %>
```

Comment:
I'm not sure I understand why this and the previously version was changed? was it previously a buggy implementation?

Reply:
Yup good point, will switch it back since this is unrelated to the changes here